govc panics when DistributedVirtualPortgroup has mac or system traffic traffic qualifier for traffic filtering and marking

```
# govc object.collect -o DistributedVirtualPortgroup:dvportgroup-81
panic: reflect.Value.Addr of unaddressable value

goroutine 1 [running]:
reflect.Value.Addr(0xf704c0, 0xc000415200, 0x16, 0x13ba300, 0xf93960, 0x16)
	/usr/local/go/src/reflect/value.go:258 +0x96
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xf93960, 0xc00026a9b0, 0x194, 0xc000415180, 0xc00026a9b0, 0x194)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:371 +0x30ac
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xed1ee0, 0xc000418088, 0x197, 0xc000415180, 0xc000418088, 0x197)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:457 +0x2834
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc000412ec0, 0x112d820, 0xc000418060, 0x199, 0x0, 0x0, 0x0, 0xc000415180, 0x1, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0x112d820, 0xc000418060, 0x199, 0xc000414e00, 0xc000418060, 0x199)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xed3ce0, 0xc000414ba0, 0x197, 0xc000414e00, 0xc000414ba0, 0x197)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:457 +0x2834
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc000412d40, 0x110bce0, 0xc000414b80, 0x199, 0x0, 0x0, 0x0, 0xc000414e00, 0x1, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xf3d780, 0xc00026daf0, 0x196, 0xc000414b40, 0xc00026daf0, 0x196)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc000412b20, 0x1039220, 0xc00026daa0, 0x199, 0x0, 0x0, 0x0, 0xc000414b40, 0x1, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0x1039220, 0xc00026daa0, 0x199, 0xc000414800, 0x199, 0x1039220)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xf938e0, 0xc00026a820, 0x194, 0xc000414800, 0xc00026a820, 0x194)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:359 +0x3024
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xed1ea0, 0xc000412a48, 0x197, 0xc000414800, 0xc000412a48, 0x197)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:457 +0x2834
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc000412a60, 0x1038640, 0xc000412a40, 0x199, 0x0, 0x0, 0x0, 0xc000414800, 0x1, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xf70340, 0xc0002131a0, 0x196, 0xc000414680, 0xc0002131a0, 0x196)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc000263960, 0x114e040, 0xc000213170, 0x199, 0x0, 0x0, 0x0, 0xc000414680, 0x203001, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0x114e040, 0xc000213170, 0x199, 0xc0004087c0, 0x199, 0x114e040)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xf892e0, 0xc0001391b0, 0x194, 0xc0004087c0, 0xc0001391b0, 0x194)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:359 +0x3024
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc000263600, 0x11883a0, 0xc000139180, 0x199, 0x0, 0x0, 0x0, 0xc0004087c0, 0x203001, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0x11883a0, 0xc000139180, 0x199, 0xc0004083c0, 0x199, 0x11883a0)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xf629c0, 0xc000210bd0, 0x194, 0xc0004083c0, 0xc000210bd0, 0x194)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:359 +0x3024
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc00000dfc0, 0x1039680, 0xc000210bc0, 0x199, 0x0, 0x0, 0x0, 0xc0004083c0, 0x203001, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0x1039680, 0xc000210bc0, 0x199, 0xc000408240, 0xc000210bc0, 0x199)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xed3de0, 0xc00007e7f0, 0x197, 0xc000408240, 0xc00007e7f0, 0x197)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:457 +0x2834
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc00000de80, 0x10f34a0, 0xc00007e7d0, 0x199, 0x0, 0x0, 0x0, 0xc000408240, 0x203001, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0x10f34a0, 0xc00007e7d0, 0x199, 0xc000265b80, 0xc00007e7d0, 0x199)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xed74a0, 0xc0002631c0, 0x197, 0xc000265b80, 0xc0002631c0, 0x197)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:457 +0x2834
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc00000de20, 0xff5760, 0xc0002631c0, 0x199, 0x0, 0x0, 0x0, 0xc000265b80, 0xc000260ad0, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xeb9800, 0xc000388b48, 0x196, 0xc000265a80, 0xc000388b48, 0x196)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc00000d9c0, 0x1097c00, 0xc000388b40, 0x199, 0x0, 0x0, 0x0, 0xc000265a80, 0xc, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0xf25140, 0xc0003956e8, 0x194, 0xc000265940, 0xc0003956e8, 0x194)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshalPath(0xc0002178c0, 0xc00000d960, 0x109b140, 0xc0003956c0, 0x199, 0x0, 0x0, 0x0, 0xc000265940, 0x0, ...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:772 +0x626
github.com/vmware/govmomi/vim25/xml.(*Decoder).unmarshal(0xc0002178c0, 0x109b140, 0xc0003956c0, 0x199, 0xc000265840, 0x199, 0x0)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:584 +0x1124
github.com/vmware/govmomi/vim25/xml.(*Decoder).DecodeElement(0xc0002178c0, 0xe8f040, 0xc0003956c0, 0x0, 0xc0002655c0, 0x224f424300251253)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:151 +0x12b
github.com/vmware/govmomi/vim25/xml.(*Decoder).Decode(...)
	/home/vmware/src/github.com/vmware/govmomi/vim25/xml/read.go:139
github.com/vmware/govmomi/vim25/soap.(*Client).RoundTrip.func1(0xc0002130e0, 0x269458f, 0x1d75700)
	/home/vmware/src/github.com/vmware/govmomi/vim25/soap/client.go:625 +0xea
github.com/vmware/govmomi/vim25/soap.(*Client).Do(0xc00012c6e0, 0x13ae580, 0xc0003ae750, 0xc0003ce400, 0xc0002516e0, 0x0, 0x0)
	/home/vmware/src/github.com/vmware/govmomi/vim25/soap/client.go:522 +0x321
github.com/vmware/govmomi/vim25/soap.(*Client).RoundTrip(0xc00012c6e0, 0x13ae500, 0xc00003a0a8, 0x139c520, 0xc000388b20, 0x139c520, 0xc000388b40, 0x1, 0x203000)
	/home/vmware/src/github.com/vmware/govmomi/vim25/soap/client.go:613 +0x91d
github.com/vmware/govmomi/vim25.(*retry).RoundTrip(0xc000262760, 0x13ae500, 0xc00003a0a8, 0x139c520, 0xc000388b20, 0x139c520, 0xc000388b40, 0x203001, 0xc000388b40)
	/home/vmware/src/github.com/vmware/govmomi/vim25/retry.go:84 +0x8a
github.com/vmware/govmomi/vim25.(*Client).RoundTrip(0xc00013a840, 0x13ae500, 0xc00003a0a8, 0x139c520, 0xc000388b20, 0x139c520, 0xc000388b40, 0x40, 0x10597a0)
	/home/vmware/src/github.com/vmware/govmomi/vim25/client.go:129 +0x80
github.com/vmware/govmomi/vim25/methods.RetrieveProperties(0x13ae500, 0xc00003a0a8, 0x139a6e0, 0xc00013a840, 0xc000395680, 0x0, 0x0, 0xc0003ae630)
	/home/vmware/src/github.com/vmware/govmomi/vim25/methods/methods.go:14219 +0xb8
github.com/vmware/govmomi/property.(*Collector).RetrieveProperties(0xc0002528d0, 0x13ae500, 0xc00003a0a8, 0x0, 0x0, 0x0, 0x0, 0xc000395640, 0x1, 0x1, ...)
	/home/vmware/src/github.com/vmware/govmomi/property/collector.go:122 +0xf6
github.com/vmware/govmomi/govc/object.(*collect).Run(0xc000210100, 0x13ae500, 0xc00003a0a8, 0xc0000747e0, 0x0, 0x0)
	/home/vmware/src/github.com/vmware/govmomi/govc/object/collect.go:412 +0x6b9
github.com/vmware/govmomi/govc/cli.Run(0xc000030090, 0x3, 0x3, 0xc000100058)
	/home/vmware/src/github.com/vmware/govmomi/govc/cli/command.go:165 +0x5de
main.main()
	/home/vmware/src/github.com/vmware/govmomi/govc/main.go:99 +0x64
```

The command works if there is only IP traffic qualifier for traffic filtering and marking.

If I reverted changes to govc/dvs/portgroup/info.go and vim25/types/if.go in https://github.com/vmware/govmomi/pull/1210, the command works.

```
# bin/govc object.collect -o DistributedVirtualPortgroup:dvportgroup-81
mo.DistributedVirtualPortgroup{
    Network: mo.Network{
        ManagedEntity: mo.ManagedEntity{
            ExtensibleManagedObject: mo.ExtensibleManagedObject{
                Self:           types.ManagedObjectReference{Type:"DistributedVirtualPortgroup", Value:"dvportgroup-81"},
                Value:          nil,
                AvailableField: nil,
            },
            Parent:              &types.ManagedObjectReference{Type:"Folder", Value:"group-n7"},
            CustomValue:         nil,
            OverallStatus:       "green",
            ConfigStatus:        "green",
            ConfigIssue:         nil,
            EffectiveRole:       []int32{-1},
            Permission:          nil,
            Name:                "",
            DisabledMethod:      nil,
            RecentTask:          nil,
            DeclaredAlarmState:  nil,
            TriggeredAlarmState: nil,
            AlarmActionsEnabled: types.NewBool(true),
            Tag:                 nil,
        },
        Summary: &types.NetworkSummary{
            Network:    &types.ManagedObjectReference{Type:"DistributedVirtualPortgroup", Value:"dvportgroup-81"},
            Name:       "dvs-trunk",
            Accessible: true,
            IpPoolName: "",
            IpPoolId:   0,
        },
        Host: []types.ManagedObjectReference{
            {Type:"HostSystem", Value:"host-20"},
            {Type:"HostSystem", Value:"host-26"},
            {Type:"HostSystem", Value:"host-32"},
        },
        Vm: []types.ManagedObjectReference{
            {Type:"VirtualMachine", Value:"vm-109"},
        },
        Name: "dvs-trunk",
    },
    Key:    "dvportgroup-81",
    Config: types.DVPortgroupConfigInfo{
        Key:                      "dvportgroup-81",
        Name:                     "dvs-trunk",
        NumPorts:                 1,
        DistributedVirtualSwitch: &types.ManagedObjectReference{Type:"VmwareDistributedVirtualSwitch", Value:"dvs-52"},
        DefaultPortConfig:        &types.VMwareDVSPortSetting{
            DVPortSetting: types.DVPortSetting{
                Blocked: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(false),
                },
                VmDirectPathGen2Allowed: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(false),
                },
                InShapingPolicy: &types.DVSTrafficShapingPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Enabled: &types.BoolPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: types.NewBool(false),
                    },
                    AverageBandwidth: &types.LongPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: 100000000,
                    },
                    PeakBandwidth: &types.LongPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: 100000000,
                    },
                    BurstSize: &types.LongPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: 104857600,
                    },
                },
                OutShapingPolicy: &types.DVSTrafficShapingPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Enabled: &types.BoolPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: types.NewBool(false),
                    },
                    AverageBandwidth: &types.LongPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: 100000000,
                    },
                    PeakBandwidth: &types.LongPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: 100000000,
                    },
                    BurstSize: &types.LongPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: 104857600,
                    },
                },
                VendorSpecificConfig: &types.DVSVendorSpecificConfig{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    KeyValue: nil,
                },
                NetworkResourcePoolKey: &types.StringPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: "-1",
                },
                FilterPolicy: &types.DvsFilterPolicy{
                    InheritablePolicy: types.InheritablePolicy{},
                    FilterConfig:      []types.BaseDvsFilterConfig{
                        &types.DvsTrafficFilterConfig{
                            DvsFilterConfig: types.DvsFilterConfig{
                                InheritablePolicy: types.InheritablePolicy{},
                                Key:               "52_81_ _-404113478",
                                AgentName:         "dvfilter-generic-vmware",
                                SlotNumber:        "",
                                Parameters:        (*types.DvsFilterParameter)(nil),
                                OnFailure:         "",
                            },
                            TrafficRuleset: &types.DvsTrafficRuleset{
                                Key:        "52_81_ _-422718066",
                                Enabled:    types.NewBool(false),
                                Precedence: 0,
                                Rules:      []types.DvsTrafficRule{
                                    {
                                        Key:         "52_81_ _-422718066_-1198708930",
                                        Description: "Network Traffic Rule 1",
                                        Sequence:    10,
                                        Qualifier:   []types.BaseDvsNetworkRuleQualifier{
                                            &types.DvsMacNetworkRuleQualifier{
                                                DvsNetworkRuleQualifier: types.DvsNetworkRuleQualifier{
                                                    Key: "52_81_ _-422718066_-1198708930_64880586",
                                                },
                                                SourceAddress:      nil,
                                                DestinationAddress: nil,
                                                Protocol:           (*types.IntExpression)(nil),
                                                VlanId:             (*types.IntExpression)(nil),
                                            },
                                        },
                                        Action:    &types.DvsUpdateTagNetworkRuleAction{},
                                        Direction: "both",
                                    },
                                    {
                                        Key:         "52_81_ _-422718066_39888505",
                                        Description: "Network Traffic Rule 2",
                                        Sequence:    20,
                                        Qualifier:   []types.BaseDvsNetworkRuleQualifier{
                                            &types.DvsIpNetworkRuleQualifier{
                                                DvsNetworkRuleQualifier: types.DvsNetworkRuleQualifier{
                                                    Key: "52_81_ _-422718066_39888505_-346040981",
                                                },
                                                SourceAddress:      nil,
                                                DestinationAddress: nil,
                                                Protocol:           &types.IntExpression{
                                                    NegatableExpression: types.NegatableExpression{
                                                        Negate: types.NewBool(false),
                                                    },
                                                    Value: 6,
                                                },
                                                SourceIpPort:      nil,
                                                DestinationIpPort: nil,
                                                TcpFlags:          (*types.IntExpression)(nil),
                                            },
                                        },
                                        Action:    &types.DvsUpdateTagNetworkRuleAction{},
                                        Direction: "both",
                                    },
                                    {
                                        Key:         "52_81_ _-422718066_-246836200",
                                        Description: "Network Traffic Rule 3",
                                        Sequence:    30,
                                        Qualifier:   []types.BaseDvsNetworkRuleQualifier{
                                            &types.DvsSystemTrafficNetworkRuleQualifier{
                                                DvsNetworkRuleQualifier: types.DvsNetworkRuleQualifier{
                                                    Key: "52_81_ _-422718066_-246836200_5716922",
                                                },
                                                TypeOfSystemTraffic: &types.StringExpression{
                                                    NegatableExpression: types.NegatableExpression{
                                                        Negate: types.NewBool(false),
                                                    },
                                                    Value: "faultTolerance",
                                                },
                                            },
                                        },
                                        Action:    &types.DvsUpdateTagNetworkRuleAction{},
                                        Direction: "both",
                                    },
                                },
                            },
                        },
                    },
                },
            },
            Vlan: &types.VmwareDistributedVirtualSwitchTrunkVlanSpec{
                VmwareDistributedVirtualSwitchVlanSpec: types.VmwareDistributedVirtualSwitchVlanSpec{},
                VlanId:                                 []types.NumericRange{
                    {
                        Start: 0,
                        End:   4094,
                    },
                },
            },
            QosTag: &types.IntPolicy{
                InheritablePolicy: types.InheritablePolicy{
                    Inherited: true,
                },
                Value: -1,
            },
            UplinkTeamingPolicy: &types.VmwareUplinkPortTeamingPolicy{
                InheritablePolicy: types.InheritablePolicy{
                    Inherited: true,
                },
                Policy: &types.StringPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: "loadbalance_srcid",
                },
                ReversePolicy: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(true),
                },
                NotifySwitches: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(true),
                },
                RollingOrder: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(false),
                },
                FailureCriteria: &types.DVSFailureCriteria{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    CheckSpeed: &types.StringPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: "minimum",
                    },
                    Speed: &types.IntPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: 10,
                    },
                    CheckDuplex: &types.BoolPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: types.NewBool(false),
                    },
                    FullDuplex: &types.BoolPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: types.NewBool(false),
                    },
                    CheckErrorPercent: &types.BoolPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: types.NewBool(false),
                    },
                    Percentage: &types.IntPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: 0,
                    },
                    CheckBeacon: &types.BoolPolicy{
                        InheritablePolicy: types.InheritablePolicy{
                            Inherited: true,
                        },
                        Value: types.NewBool(false),
                    },
                },
                UplinkPortOrder: &types.VMwareUplinkPortOrderPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    ActiveUplinkPort:  []string{"uplink1", "uplink2", "uplink3", "uplink4"},
                    StandbyUplinkPort: nil,
                },
            },
            SecurityPolicy: &types.DVSSecurityPolicy{
                InheritablePolicy: types.InheritablePolicy{
                    Inherited: true,
                },
                AllowPromiscuous: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(false),
                },
                MacChanges: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(false),
                },
                ForgedTransmits: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(false),
                },
            },
            IpfixEnabled: &types.BoolPolicy{
                InheritablePolicy: types.InheritablePolicy{
                    Inherited: true,
                },
                Value: types.NewBool(false),
            },
            TxUplink: &types.BoolPolicy{
                InheritablePolicy: types.InheritablePolicy{
                    Inherited: true,
                },
                Value: types.NewBool(false),
            },
            LacpPolicy: &types.VMwareUplinkLacpPolicy{
                InheritablePolicy: types.InheritablePolicy{
                    Inherited: true,
                },
                Enable: &types.BoolPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: types.NewBool(false),
                },
                Mode: &types.StringPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Value: "passive",
                },
            },
            MacManagementPolicy: &types.DVSMacManagementPolicy{
                InheritablePolicy: types.InheritablePolicy{
                    Inherited: true,
                },
                AllowPromiscuous:  types.NewBool(false),
                MacChanges:        types.NewBool(false),
                ForgedTransmits:   types.NewBool(false),
                MacLearningPolicy: &types.DVSMacLearningPolicy{
                    InheritablePolicy: types.InheritablePolicy{
                        Inherited: true,
                    },
                    Enabled:              false,
                    AllowUnicastFlooding: (*bool)(nil),
                    Limit:                (*int32)(nil),
                    LimitPolicy:          "",
                },
            },
            VNI: &types.IntPolicy{
                InheritablePolicy: types.InheritablePolicy{
                    Inherited: true,
                },
                Value: 0,
            },
        },
        Description: "VMware Integrated OpenStack: Neutron DVS Plugin for neutron1",
        Type:        "ephemeral",
        BackingType: "standard",
        Policy:      &types.VMwareDVSPortgroupPolicy{
            DVPortgroupPolicy: types.DVPortgroupPolicy{
                BlockOverrideAllowed:               true,
                ShapingOverrideAllowed:             false,
                VendorConfigOverrideAllowed:        false,
                LivePortMovingAllowed:              false,
                PortConfigResetAtDisconnect:        true,
                NetworkResourcePoolOverrideAllowed: types.NewBool(false),
                TrafficFilterOverrideAllowed:       types.NewBool(false),
            },
            VlanOverrideAllowed:           false,
            UplinkTeamingOverrideAllowed:  false,
            SecurityPolicyOverrideAllowed: false,
            IpfixOverrideAllowed:          types.NewBool(false),
            MacManagementOverrideAllowed:  types.NewBool(false),
        },
        PortNameFormat:               "",
        Scope:                        nil,
        VendorSpecificConfig:         nil,
        ConfigVersion:                "10",
        AutoExpand:                   types.NewBool(false),
        VmVnicNetworkResourcePoolKey: "-1",
        Uplink:                       types.NewBool(false),
        TransportZoneUuid:            "",
        TransportZoneName:            "",
        LogicalSwitchUuid:            "",
        SegmentId:                    "",
    },
    PortKeys: []string{"23"},
}
```

I use the latest code pulling from github.

--- a/govc/dvs/portgroup/info.go
+++ b/govc/dvs/portgroup/info.go
@@ -214,15 +214,22 @@ func (r *infoResult) Write(w io.Writer) error {
 				setting.FilterPolicy.FilterConfig != nil &&
 				len(setting.FilterPolicy.FilterConfig) > 0 {
 
-				rules := setting.FilterPolicy.FilterConfig[0].GetDvsTrafficFilterConfig()
-				if rules != nil && rules.TrafficRuleset != nil && rules.TrafficRuleset.Rules != nil {
+				rules, ok := setting.FilterPolicy.FilterConfig[0].(*types.DvsTrafficFilterConfig)
+				if !ok {
+					continue
+				}
+				if rules != nil && rules.TrafficRuleset != nil {
 					for _, rule := range rules.TrafficRuleset.Rules {
 						for _, q := range rule.Qualifier {
 							var protocol string
-							if val, ok := protocols[q.GetDvsIpNetworkRuleQualifier().Protocol.Value]; ok {
+							ipr, ok := q.(*types.DvsIpNetworkRuleQualifier)
+							if !ok {
+								continue
+							}
+							if val, ok := protocols[ipr.Protocol.Value]; ok {
 								protocol = val
 							} else {
-								protocol = fmt.Sprintf("%d", q.GetDvsIpNetworkRuleQualifier().Protocol.Value)
+								protocol = fmt.Sprintf("%d", ipr.Protocol.Value)
 							}
 
 							trafficRuleSet[portID][int(rule.Sequence)] = trafficRule{
@@ -230,10 +237,10 @@ func (r *infoResult) Write(w io.Writer) error {
 								Direction:          rule.Direction,
 								Action:             printAction(rule.Action),
 								Protocol:           protocol,
-								SourceAddress:      printAddress(q.GetDvsIpNetworkRuleQualifier().SourceAddress),
-								SourceIpPort:       printPort(q.GetDvsIpNetworkRuleQualifier().SourceIpPort),
-								DestinationAddress: printAddress(q.GetDvsIpNetworkRuleQualifier().DestinationAddress),
-								DestinationIpPort:  printPort(q.GetDvsIpNetworkRuleQualifier().DestinationIpPort),
+								SourceAddress:      printAddress(ipr.SourceAddress),
+								SourceIpPort:       printPort(ipr.SourceIpPort),
+								DestinationAddress: printAddress(ipr.DestinationAddress),
+								DestinationIpPort:  printPort(ipr.DestinationIpPort),
 							}
 						}
 					}

--- a/vim25/types/if.go
+++ b/vim25/types/if.go
@@ -798,7 +798,6 @@ func (b *DvsFilterConfig) GetDvsFilterConfig() *DvsFilterConfig { return b }
 
 type BaseDvsFilterConfig interface {
 	GetDvsFilterConfig() *DvsFilterConfig
-	GetDvsTrafficFilterConfig() *DvsTrafficFilterConfig
 }
 
 func init() {
@@ -841,19 +840,10 @@ func (b *DvsNetworkRuleQualifier) GetDvsNetworkRuleQualifier() *DvsNetworkRuleQu
 
 type BaseDvsNetworkRuleQualifier interface {
 	GetDvsNetworkRuleQualifier() *DvsNetworkRuleQualifier
-	GetDvsIpNetworkRuleQualifier() *DvsIpNetworkRuleQualifier
 }
 
 func init() {
 	t["BaseDvsNetworkRuleQualifier"] = reflect.TypeOf((*DvsNetworkRuleQualifier)(nil)).Elem()
-}
-
-func (b *DvsIpNetworkRuleQualifier) GetDvsIpNetworkRuleQualifier() *DvsIpNetworkRuleQualifier {
-	return b
-}
-
-type BaseDvsIpNetworkRuleQualifier interface {
-	GetDvsIpNetworkRuleQualifier() *DvsIpNetworkRuleQualifier
 }
 
 func (b *DvsTrafficFilterConfig) GetDvsTrafficFilterConfig() *DvsTrafficFilterConfig { return b }

